components/modal/styles.css

```diff
@@ -0,0 +1,44 @@
+:root {
+    --primary-color: #4a4b4b;
+    --second-color: #9a16ce;
+    --third-color: #c65efa;
+    --white: rgb(231, 227, 227);
+}
+.modal-overlay {
+    position: absolute;
+    filter: blur(2.22px) grayscale(0.46);
+    height: 100vh;
+    width: 100vw;
```

Comment:
I think because of this horizontal scroll bar is coming, you can remove this

Reply:
thanks, viresh this one is resolved.